databuilder/models/dashboard/dashboard_table.py

```diff
@@ -54,7 +54,7 @@ def create_next_relation(self):
     def _create_relation_iterator(self):
         # type: () -> Optional[None, Iterator[[Dict[str, Any]]]]
         for table_id in self._table_ids:
-            m = re.match('(\w+)://(\w+)\.(\w+)\/(\w+)', table_id)
+            m = re.match(r'(\w+)://(\w+)\.(\w+)\/([\w.]+)', table_id)
```

Comment:
What's this about? Is it the table names with 3 parts? eg `bi_reporting.bi_reporting.table`

Reply:
Yep, that's exactly what it's for. I implemented this as a workaround solution until we could figure out the real reason for the double schema, it was breaking the table insertion process and preventing me from moving forward so I just added this quick hack. We should definitely address the root cause, once that's done this can probably be reverted.